src/DnsUtils.ts

```diff
@@ -0,0 +1,106 @@
+import DnsRecordsError, { DnsRecordsErrorCode } from "./errors/dnsRecordsError";
+import { CryptoRecords, DnsRecord, DnsRecordType } from "./publicTypes";
+import { isStringArray } from "./utils";
+export default class DnsUtils {
+
+  static readonly DEFAULT_TTL: number = 300; // 5 minutes 
```

Comment:
Use `DefaultTTL`. In our convension snake case should never be used.

Reply:
I'd vote for `DefaultTtl`

https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN

> When using acronyms, use Pascal case or camel case for acronyms more than two characters long. For example, use `HtmlButton` or `htmlButton`. However, you should capitalize acronyms that consist of only two characters, such as `System.IO` instead of `System.Io`.